sct_tutorial/SCT_Tutorial_01_Segmentation.ipynb

```diff
@@ -0,0 +1,445 @@
+{
```

Comment:
## Alternate proposal: A single notebook with commands only

As a way to keep all of the "pros" of this idea, but mitigate the "cons", what if we just had a single notebook containing no theory, and instead just the commands as well as section titles? 

Because it would be barebones and simplified, it would be much easier to maintain, and effectively serve the same purpose as the existing [`batch_single_subject.sh`](https://github.com/spinalcordtoolbox/sct_tutorial_data/blob/master/single_subject/batch_single_subject.sh) script in `sct_tutorial_data`. It could even replace that script!

That way, instead of copying and pasting commands from the script into the terminal (like we did in the last SCT Course), we would execute the commands one by one directly in the notebook. But, the overall flow of the course would stay the same (using the slide deck, pausing for "hands-on" portions, etc).